src/main/java/org/openstreetmap/atlas/geography/atlas/change/validators/ChangeValidator.java

```diff
@@ -50,6 +58,193 @@ protected void validateChangeNotEmpty()
         }
     }
 
+    protected void validateGeometricRelationsUpdated()
+    {
+        this.change.changesFor(ItemType.EDGE)
```

Comment:
Woooooo maybe break this one in more methods?

Reply:
yeah this is on the todo list lol